docs/user-docs/latest/settings/managing-global-settings.md

```diff
@@ -131,3 +131,10 @@ To create a new API token:
 ::: caution
 For security reasons, API tokens are only shown right after they have been created. When refreshing the page or navigating elsewhere in the admin panel, the newly created API token will be hidden and will not be displayed again.
 :::
+
+## Managing settings for custom fields
+
+[Custom fields](/user-docs/latest/plugins/introduction-to-plugins.md#custom-fields) can add their own settings sub-sections. These sections are found at the bottom of the list of sub-sections, below settings for Strapi built-in plugins.
```

Comment:
This could be a little bit confusing: it's the plugin that creates the settings, not the custom field itself. So via the plugin API, a single plugin can register at the same a custom field, and a page of settings that may impact how the custom field is used

Reply:
Thanks, indeed! I was confusing things.

So, actually, what we have in [this Figma screenshot](https://www.figma.com/file/Ij1vIaGuxetkrmwAbTFVKw/%5B%F0%9F%8F%97WIP%5D-Expansion---Custom-fields?node-id=2509%3A30973) is not really something new for custom fields. It's only because the [Settings API](https://docs.strapi.io/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.html#settings-api) was used by a plugin that we have a new settings section, right?

Reply:
Yes exactly, it's not a new API, it's just highlighting how different parts of the plugin API can work together

Reply:
Awesome, thanks. I'll update this section in light of your comment.